submissions/DarkSoulH/Hooli-style Popup/index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+
+    <link rel="stylesheet" href="/Hooli-style-Popup/assets/css/style.css">
+
+</head>
+<body>
+
+<div class="case">
+    <div class="header">
+        <div class="container">
+            <nav class="top_bar">
+                <ul class="top_menu">
+                    <li class="menu_item">
+                        <a class="menu_item_link" href="#">Mail</a>
+                    </li>
+
+                    <li class="menu_item">
+                        <a class="menu_item_link" href="#">Images</a>
+                    </li>
+
+                    <li class="menu_item">
+                        <input id="checkbox" class="checkbox" type="checkbox">
```

Comment:
[Watch video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide checkbox in the right way.